vcd/resource_vcd_vapp_vm.go

```diff
@@ -238,6 +210,81 @@ func resourceVcdVAppVmCreate(d *schema.ResourceData, meta interface{}) error {
 	return resourceVcdVAppVmUpdate(d, meta)
 }
 
+// Adds existing org VDC network to VM network configuration
+func addVdcNetwork(d *schema.ResourceData, vdc govcd.Vdc, vapp govcd.VApp, vcdClient *VCDClient) ([]*types.OrgVDCNetwork, string, error) {
```

Comment:
What is this function returning?
Either add the definition of the return values in a comment or use named return types to show what we are returning.
For example:
```
func addVdcNetwork(d *schema.ResourceData, vdc govcd.Vdc, vapp govcd.VApp, vcdClient *VCDClient) (vdcNetworks []*types.OrgVDCNetwork, networkName string, err error) 
```

Reply:
Fixed